docs/UserGuide.md

```diff
@@ -316,11 +316,15 @@ Format: `addClient n/NAME p/PHONE_NUMBER [a/ADDRESS] [e/EMAIL] [b/BIRTHDAY] [pd/
 * A client **must** have a `NAME` and a `PHONE_NUMBER`.
 * `PHONE_NUMBER` should contain only numbers and be at least 8 digits long.
 * `EMAIL`, `BIRTHDAY`, `ADDRESS` and `PRODUCT` are optional.
-* If a `NAME` already exist in _MyInsuRec_, adding the same `NAME` will result in an error!
+* If a `NAME` already exists in _MyInsuRec_, adding the same `NAME` will result in an error!
 
 Use case:
 1. You have just found a potential client! You can use this command to add their particulars into _MyInsuRec_ to help remember their contact information and other details.
 
+<div markdown="span" class="alert alert-success">**:bulb: Tips and tricks:**
+We are using `NAME` as a unique identifier because we have considered cases where parents will be using their contact details for their children. If you are stuck in a situation where both clients have the same name, you can add a few strings to make them unique, for example John Tan 1 and John Tan 2.
```

Comment:
maybe not `add a few strings` a bit foreign to people to don't code. Maybe can say `add a few more words`? and the example maybe can use `John Tan (NUS)` and `John Tan (SMU)` rather than just using 1 and 2